packages/neon/neon/lib/neon.dart

```diff
@@ -36,11 +35,11 @@ Future runNeon({
 
   final sharedPreferences = sharedPreferencesOverride ?? await SharedPreferences.getInstance();
 
-  final platform = await getNeonPlatform();
-  final cache = Cache(platform);
+  await NeonPlatform.setup();
+  final cache = Cache();
   await cache.init();
-  final requestManager = RequestManager(cache);
-  final allAppImplementations = getAppImplementations(sharedPreferences, requestManager, platform);
+  RequestManager.cache = cache;
```

Comment:
the Cache is only used  by `RequestManager` thus the requestmanager should handle the setup of the cache.

The cache should also be marked as internal and hidden from the export.